src/sentry/api/endpoints/group_ai_autofix.py

```diff
@@ -111,11 +111,6 @@ def _call_autofix(
                     else None
                 ),
                 "options": {
-                    "disable_codebase_indexing": features.has(
-                        "organizations:autofix-disable-codebase-indexing",
```

Comment:
are we removing these flags elsewhere in another PR?

Reply:
yes following up with a flag removal pr